internal/configparse/json_test.go

```diff
@@ -109,3 +115,11 @@ func sameErrors(a, b error) bool {
 	}
 	return a.Error() == b.Error()
 }
+
+func TestMap(t *testing.T) {
+	base := object{"a": "a"}
+	base.assign(object{"a": "CHANGED"})
+	if base["a"] != "a" {
+		t.Errorf("base map was mutated")
+	}
+}
```

Comment:
Do we use this somewhere?

Reply:
Bad-faith answer: technically we use it, because all `TestXxx` funcs are run in the test suite.
Honest answer: ...but obviously it has nothing to do here 😅 I made it to check whether maps were always passed by reference (they are!) and forgot to remove it. Good catch!